core/templates/dev/head/pages/exploration-editor-page/statistics-tab/issues/answer-submit-action.directive.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the Answer Submit Learner Action.
+ */
+
+require('domain/utilities/UrlInterpolationService.ts');
+require('services/ExplorationHtmlFormatterService.ts');
+require('services/HtmlEscaperService.ts');
+
+var oppia = require('AppInit.ts').module;
+
+oppia.directive('answerSubmitAction', [
+  'ExplorationHtmlFormatterService', 'HtmlEscaperService',
+  'UrlInterpolationService',
+  function(
+      ExplorationHtmlFormatterService, HtmlEscaperService,
+      UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {},
+      bindToController: {},
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/exploration-editor-page/statistics-tab/issues/' +
+        'answer-submit-action.directive.html'),
+      controllerAs: '$ctrl',
+      controller: ['$attrs', function($attrs) {
+        var ctrl = this;
+        ctrl.currentStateName = $attrs.currentstatename;
+        ctrl.destStateName = $attrs.deststatename;
```

Comment:
Should be $attrs.destStateName etc. (This happens because you should be using hyphens for the attribute names.)

Reply:
Done.